docs/companion.md

```diff
@@ -563,6 +563,17 @@ app.use(
 When signing on the client, this function will only be called for multipart
 uploads.
 
+##### `s3.acl`
+
+Specify the
+[Canned ACL](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl).
+
+##### `s3.serverSideEncryption`
+
+Specify the
+[Server-Side Encryption](https://docs.aws.amazon.com/AmazonS3/latest/dev/serv-side-encryption.html)
```

Comment:
should we maybe also link to https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-client-s3/Variable/ServerSideEncryption/ - so the user knows which values are valid?

also I think we need a corresponding env variable. maybe `COMPANION_AWS_S3_SERVER_SIDE_ENCRYPTION`